.github/dependabot.yml

```diff
@@ -79,6 +79,7 @@ updates:
       - dependency-name: io.fabric8:docker-maven-plugin
       - dependency-name: net.revelc.code.formatter:formatter-maven-plugin
       - dependency-name: net.revelc.code:impsort-maven-plugin
+      - dependency-name: org.apache.maven.plugins:maven-invoker-plugin
```

Comment:
I do think we should start adding more standard Maven plugins.
Even for compiler-plugin, which often requires additional adjustments for codestarts, it's still worthwhile to know when we are falling behind.

Reply:
Sure, why not?

Reply:
That would be a separate PR then 🙂

Reply:
Of course 😉